### ISSUE TITLE
Rsync mirror server

### ISSUE THREAD

**Issue by GothAck** [nexus511/gpd-ubuntu-packages#31]
I have a European mirror server that I'm spinning up & am happy to add the packages & isos to. Could we set up a periodic rsync a la how ubuntu / debian mirrors are synced?

**Comment by nexus511** [comment]
@GothAck Yes. I already have a rsync-push job running when a new release is available.

We could also add a mirror for the packages but as those are pretty small, that is not a problem for now.

**Comment by GothAck** [comment]
I can set up an ssh account you can push to, or I can add a periodic sync script to poll a read-only rsyncd as I currently do for raspbian; either is cool, but the latter is preferable as I usually keep my boxes very locked down. As for the packages  ¯\_(ツ)_/¯ if we're at it I'm not opposed to mirroring everything.
